### PR TITLE
add **kwargs to cgo_genrule

### PR DESCRIPTION
**Pull request by mikedanese**
This allows additions of tags attrs and such and will improve backwards compatibility of the rule.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by pmbethe09** [comment]
Jenkins, test this please.

File: go/def.bzl [hunk 1]
```diff
@@ -982,7 +982,8 @@ def _setup_cgo_library(name, srcs, cdeps, copts, clinkopts, go_tool, toolchain):
 def cgo_genrule(name, srcs,
                 copts=[],
                 clinkopts=[],
-                cdeps=[]):
+                cdeps=[],
+                **kwargs):
```
Comment (pmbethe09) [inline go/def.bzl]: don't you need to then pass this down to some other call?

Comment (mikedanese) [inline go/def.bzl]: Yup, thanks.

**Comment by mikedanese** [comment]
@pmbethe09 Done. Please take a look.

**Comment by pmbethe09** [comment]
Jenkins, test this please.

**Pull request merged**